UA Core Library.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.26430.16
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.29806.167
```

Comment:
Please revert this, so we Visual Studio 15 as in all other .sln files

Reply:
I'm working on that. - Missed that one.

Reply:
Fixed and submitted.